translate/PT.json

```diff
@@ -333,64 +333,64 @@
+        ["buybase.extended", "A base %s foi renovada com sucesso! O tempo restante é de %s dias."],
+        ["basehint.main", "%s Base\nPreço: %s BC / 30 dias\n\n%s\n\nClique para obter mais informações e ações."],
+        ["basehint.available", "Esta base está disponível para aluguel!"],
+        ["basehint.busy", "Esta base está ocupada até %s"],
+        ["basehint.soon", "Em alguns dias, provavelmente estará disponível."],
         ["baseinfo.title", "%s Base"],
-        ["baseinfo.available", "Available for rent!"],
-        ["baseinfo.busy", "Busy until %s"],
-        ["baseinfo.props", "Location: %s\nPlayer slots: %s\nVehicle slots: %s\nHelicopter slots: %s"],
-        ["baseinfo.props.noslots", "None"],
-        ["baseinfo.rent", "Rent for %s days"],
-        ["baseinfo.extend", "Prolongation for %s days"],
-        ["baseinfo.close", "Close"],
-        ["baseinfo.back", "Back"],
-        ["basepage.title", "Base %s info on MTA DayZ Ultimate \"%s\""],
-        ["basepage.cmdcontrol", "Your commands for control base:%s"],
-        ["basepage.cmdmanager", "Command for open base manager panel:%s"],
-        ["basepage.cmdclean", "Command for manual cleaning vehicles in base:%s"],
-        ["basepage.ownercommands", "Be careful!\nOnly base owner can open control panel and clean base vehicles."],
-        ["basepage.bind", "Binds guide:%s"],
-        ["basepage.unbind", "How to unbind key:%s"],
-        ["basepage.other", "... and other keys."],
-        ["basegate.main", "Main Gate%s."],
-        ["basegate.secondary", "Secondary Gate%s."],
-        ["basegate.helipad", "Helipad%s."],
-        ["basegate.heligate", "Heligate%s."],
-        ["basegate.elevator", "Elevator%s."],
-        ["coupon.title", "Coupon activation."],
-        ["coupon.desc", "Enter your coupon code in the text box below.\nWhen you've entered it, click activate button, and\nget various gifts!"],
-        ["coupon.titlefield", "Enter coupon:"],
-        ["coupon.activate", "Activate"],
-        ["coupon.back", "Back"],
+        ["baseinfo.available", "Disponível para aluguel!"],
+        ["baseinfo.busy", "Ocupada até %s"],
+        ["baseinfo.props", "Localização: %s\nVagas de jogador: %s\nVagas de veículo: %s\nVagas de helicóptero: %s"],
+        ["baseinfo.props.noslots", "Nenhum"],
+        ["baseinfo.rent", "Alugar por %s dias"],
+        ["baseinfo.extend", "Prolongamento por %s dias"],
+        ["baseinfo.close", "Fechar"],
+        ["baseinfo.back", "Voltar"],
+        ["basepage.title", "Informações da base %s em MTA DayZ Ultimate \"%s\""],
+        ["basepage.cmdcontrol", "Seus comandos para controlar a base:%s"],
+        ["basepage.cmdmanager", "Comando para abrir o painel do gerenciador da base:%s"],
+        ["basepage.cmdclean", "Comando para limpar manualmente os veículos na base:%s"],
+        ["basepage.ownercommands", "Tenha cuidado!\nSomente o proprietário da base pode abrir o painel de controle e limpar os veículos da base.""],
```

Comment:
Double " in ends of this line is break JSON file structure. Should be (**da base."],**) instead (**da base.""],**).